objective-c/helloworld/README.md

```diff
@@ -0,0 +1,50 @@
+#gRPC in 3 minutes (Objective-C)
+
+## Installation
+
+To run this example you should have [Cocoapods](https://cocoapods.org/#install) installed, as well as the relevant tools to generate the client library code (and a server in another language, for testing). You can obtain the latter by following [these setup instructions](https://github.com/grpc/homebrew-grpc).
+
+## Hello Objective-C gRPC!
+
+Here's how to build and run the Objective-C implementation of the [Hello World](https://github.com/grpc/grpc-common/blob/master/protos/helloworld.proto) example used in [Getting started](https://github.com/grpc/grpc-common).
```

Comment:
I wouldn't point people to the grpc-common README as it is now: It's super long, and pretty Java- and Go- specific. Even though we're not telling them to read it first, we're kind of implying that familiarity with what's said there will be useful.